Makefile

```diff
@@ -25,13 +25,19 @@ deps:
 		false; \
 	fi
 	curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+	curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+	sudo add-apt-repository \
+		"deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+		`lsb_release -cs` \
+		stable"
 	-sudo apt-get -qq update
 	if [ `lsb_release -rs` = "14.04" ]; then \
```

Comment:
... don't bother with this one, because at this point the standard name is docker-engine for all versions.